modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -300,12 +301,29 @@ protected ObjectDefinition randomObjectDefinition() throws Exception {
 						indexed = false;
 						indexedAsKeyword = false;
 						label = Collections.singletonMap("en_US", "Column");
+						localized =
+							objectDefinition.getEnableLocalization() &&
+							!objectDefinition.getSystem() &&
+							RandomTestUtil.randomBoolean();
 						name = StringUtil.randomId();
 						required = false;
 						system = false;
 					}
 				}
 			});
+		objectDefinition.setStatus(
+			new Status() {
+				{
+					code = WorkflowConstants.STATUS_DRAFT;
+					label = WorkflowConstants.getStatusLabel(
+						WorkflowConstants.STATUS_DRAFT);
+					label_i18n = _language.get(
+						LanguageResources.getResourceBundle(
+							LocaleUtil.getDefault()),
+						WorkflowConstants.getStatusLabel(
+							WorkflowConstants.STATUS_DRAFT));
+				}
+			});
```

Comment:
@seltong why the status needs to be DRAFT?

Reply:
Now we are looking at status.

```
@Override
protected String[] getAdditionalAssertFieldNames() {
	return new String[] {"name", "status"};
}
```

Therefore, we need to ensure that each created objectDefinition has the correct status and the default status is Draft.